api/routers/odin.py

```diff
@@ -83,14 +87,17 @@ async def rollback_service(rollback_request: RollbackRequest):
 
 
 @router.get("/odin/services/", tags=["odin"])
-async def get_all_services():
+async def get_all_services(user_id: str):
     try:
-        service_list = Utils.getJson(Helm.listAllServices())
+        if Utils.checkIfKeyExist(etcd, '{user_id}:service_list'.format(user_id=user_id)):
```

Comment:
'user_id' is not used in user management. 
Use the acl-token instead.

Like 'acl:acl_token-{the_actual_acl_token}' : [list of services]

Aisa mapping maitain kar when inserting or search for an object.

Reply:
@TheAlgo

Reply:
Correction: keep it like this "service:acl_token-{actual-acl-token}"

Reply:
I don't think we need to be that verbose instead we can keep it as `{acl_token}:service_list`